package.json

```diff
@@ -1,17 +1,26 @@
 {
-  "dependencies": {
-    "@babel/polyfill": "^7.0.0",
-    "chart.js": "^3.7.1",
-    "jquery": "^3.5.0",
-    "jquery.dirtyforms": "^2.0.0",
-    "lightbox2": "^2.9.0",
-    "semantic-ui-css": "^2.2.0",
-    "slick-carousel": "^1.8.1"
+  "name": "@sylius-ui/frontend",
+  "version": "1.0.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Sylius/Sylius.git"
   },
-  "devDependencies": {
+  "license": "MIT",
+  "author": "Paweł Jędrzejewski",
```

Comment:
Should it be?
```suggestion
  "author": ""Sylius Sp. z o.o.",
```

Reply:
`@sylius-ui/frontend` is a meta package. The end project requires all those deps, and `@sylius-ui/frontend` should be required in the dev deps section in a Sylius end-app.